frontstage/controllers/survey_controller.py

```diff
@@ -0,0 +1,24 @@
+import json
+import logging
+
+from flask import current_app as app
+from structlog import wrap_logger
+
+from frontstage.common.request_handler import request_handler
+from frontstage.exceptions.exceptions import ApiError
+
+
+logger = wrap_logger(logging.getLogger(__name__))
+
+
+def get_survey(survey_id):
```

Comment:
This should be removed as well as it's related to my comment above.  Moving survey service code should happen in a later PR.

Reply:
@insacuri this controller is not used by these features, I'll get it removed from this branch 👍